models/stg_twilio.yml

```diff
@@ -349,8 +349,8 @@ models:
     tests: 
       - dbt_utils.unique_combination_of_columns:
           combination_of_columns:
-            - account_id
-            - messaging_service_id
+            - account_id 
+            - messaging_service_id 
```

Comment:
Same not null test comment

Reply:
added not null and unique test to messaging_service_id , and not null test to account_id for `stg_twilio__messaging_service`

Reply:
Why is the unique test being added to messaging_service_id? If that is the case, why are we also doing a unique combo of columns test?

Reply:
True, same logic applies as for `stg_twilio__message`. Removed the unique test for just messaging_service_id